### ISSUE TITLE
Package as extension

### ISSUE THREAD

**Issue by Tostino** [christopherthompson81/pgsql_holidays#5]
Have you thought about packaging this as an extension for Postgres?

Doing so can make distribution easier, and increase uptake.
You also can benefit from publishing this as an extension on https://pgxn.org/ for example.

**Comment by christopherthompson81** [comment]
I have. I suppose my only hesitation is that PGXN seems obscure even though its the only extension repository I can see. Is it in prevalent use?

**Comment by Tostino** [comment]
Yeah pgxn is pretty widely used as far as things go, and the more that is released on it the better. Even without releasing it through pgxn, it is nice to use the standard Postgres packaging for this type of feature (an extension), as they have ways to manage version upgrades, etc.

**Comment by madtibo** [comment]
@Tostino I added a makefile to build pgsql_holidays as a PostgreSQL extension: https://github.com/christopherthompson81/pgsql_holidays/pull/13
